pkgs/development/python-modules/dask-yarn/default.nix

```diff
@@ -0,0 +1,29 @@
+{ buildPythonPackage
+, dask
+, distributed
+, fetchPypi
+, grpcio
+, grpcio-tools
+, pytest
+, skein
+, stdenv
+}:
+
+buildPythonPackage rec {
+  version = "0.8.1";
+  pname = "dask-yarn";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1vdlzk3sgi77hpfl3k89ln6qk2m6zqb8hdfcrgw08q6awb97rfj4";
+  };
```

Comment:
Would it be possible to fetchFromGitHub instead? Usually that way you get more tests etc

Reply:
anyway please also add a pythonImportsCheck.